functions/queries.py

```diff
@@ -31,6 +31,11 @@ def get_nests(event):
 def get_story(event):
     try:
         story = Story.get(event["arguments"]["nestId"], f"STORY.{event['arguments']['storyId']}")
+
+        # Convert completedAt property to a string
+        if story.completedAt:
+            story.completedAt = story.completedAt.strftime('%Y-%m-%d')
```

Comment:
We shouldn't be handling this any different from any other datetime field.  Should be able to handle this for the frontend.  If we were to do this in the backend it should be handled in the `to_dict` funciton in the story model.